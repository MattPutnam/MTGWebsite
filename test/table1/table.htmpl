<table class="{{eval('$header'.replace(' ', '_'))}}">
    <thead>
        <tr>
            <th colspan="2">{{$header}}</th>
        </tr>
    </thead>
    <tbody>{{foreach:var=row, source=$content}}
        <tr>
            <td class="role">{{$row/role}}</td>
            <td class="name">{{$row/name}}</td>
        </tr>{{end}}
    </tbody>
</table>