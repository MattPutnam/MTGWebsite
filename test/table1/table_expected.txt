<table class="Test_Table">
    <thead>
        <tr>
            <th colspan="2">Test Table</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td class="role">Producer</td>
            <td class="name">Alice</td>
        </tr>
        <tr>
            <td class="role">Director</td>
            <td class="name">Bob</td>
        </tr>
    </tbody>
</table>